pkg/analytics/fields_listener.go

```diff
@@ -45,38 +45,37 @@ func (fl *fieldListener) Write(entry zapcore.Entry, fields []zapcore.Field) erro
 	allFields = append(allFields, fl.fields...)
 	allFields = append(allFields, fields...)
 
-	message := entry.Level.CapitalString()
+	logLevel := Warn
+	switch entry.Level {
+	case zapcore.DebugLevel:
+		logLevel = Debug
+	case zapcore.InfoLevel:
+		logLevel = Info
+	case zapcore.WarnLevel:
+		logLevel = Warn
+	case zapcore.ErrorLevel:
+		logLevel = Error
+	case zapcore.DPanicLevel, zapcore.PanicLevel, zapcore.FatalLevel:
+		logLevel = Panic
+	}
+
+	p := fl.client.createPayload(logLevel, entry.Level.CapitalString())
```

Comment:
Shouldn't the event be the same (like "log") since the level is already in the properties? Seems odd to have this duplicated is all

Reply:
I think I prefer this a bit, just because it makes it a bit easier as you're scanning the all-events dashboard to visually pick out warnings vs logs.